haas/test_common.py

```diff
@@ -273,6 +273,15 @@ def collect_nodes(self):
                 len(nodes))
         return nodes
 
+    def get_config(self, config_type):
```

Comment:
This has problems:

1. It leaks the network connection.
2. It calls a private method specific to the dell driver, without any guarantee that the switch uses that driver.